src/backend/app/config.py

```diff
@@ -116,6 +116,9 @@ def assemble_db_connection(cls, v: Optional[str], info: ValidationInfo) -> Any:
     OSM_URL: HttpUrlStr = "https://www.openstreetmap.org"
     OSM_SCOPE: str = "read_prefs"
     OSM_LOGIN_REDIRECT_URI: str = "http://127.0.0.1:7051/osmauth/"
+    OSM_SVC_TOKEN: str = """ZXlKcFpDSTZNakF6T0RZeU1Ua3NJblZ6WlhKdVlXMWx
```

Comment:
This shouldn't be hardcoded, but configurable.

In the Pydantic, specifying `Optional[str]` makes the field nullable. Specifying `Optional[str] = None` makes the field optional (what we want).

The value is then injected via the `.env` file.
Don't worry about updating the example dotenv etc, I can do that after.